mslib/msui/_tests/test_mscolab_version_history.py

```diff
@@ -133,8 +133,8 @@ def test_refresh(self):
         assert new_changes_count == changes_count + 2
 
     def _connect_to_mscolab(self):
-        self.window.url.setEditText("http://localhost:8084")
-        QtTest.QTest.mouseClick(self.window.connectMscolab, QtCore.Qt.LeftButton)
+        self.window.url.setEditText("http://localhost:8083")
```

Comment:
the port is wrong, see for comparison 
https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/_tests/test_mscolab_version_history.py#L136

We try to avoid 8083 because this can get easily in conflict with a developer server

Reply:
Alright, I will resolve this